docker/gazebo/gzsource/Dockerfile

```diff
@@ -17,8 +17,18 @@ RUN bash -c 'source /etc/os-release && \
 RUN curl https://xpra.org/gpg.asc | apt-key add -
 RUN curl http://packages.osrfoundation.org/gazebo.key | apt-key add -
 RUN apt-get update && apt-get install -y xpra=2.3.*
-RUN apt-get update && apt-get install -y gazebo$GZV
-RUN apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+RUN apt-get update && apt-get install -y build-essential cmake debhelper mesa-utils cppcheck xsltproc python-lxml python-psutil python bc netcat-openbsd gnupg2 net-tools locales
+RUN apt-get update && apt-get install -y libfreeimage-dev libprotoc-dev libprotobuf-dev protobuf-compiler freeglut3-dev libcurl4-openssl-dev libtinyxml-dev libtar-dev libtbb-dev libogre-1.9-dev libxml2-dev pkg-config qtbase5-dev libqwt-qt5-dev libltdl-dev libgts-dev libboost-thread-dev libboost-signals-dev libboost-system-dev libboost-filesystem-dev libboost-program-options-dev libboost-regex-dev libboost-iostreams-dev libbullet-dev libsimbody-dev libignition-common-dev libignition-fuel-tools-dev libignition-transport4-dev libignition-math4-dev libignition-msgs-dev libtinyxml2-dev libsdformat6-dev
```

Comment:
these are probably the dependencies for gazebo9. If we hardcoded them here directly, how are we going to handle the different versions of Gazebo supported?

Also please note that the .repos file will bring the sources for ignition, sdformat and gazebo so we probably don't want to have the binary packages installed.

Reply:
> these are probably the dependencies for gazebo9. If we hardcoded them here directly, how are we going to handle the different versions of Gazebo supported?

Ouch just read the TODO item list. The second comment of binary packages is still valid though.

Reply:
Yes, I had noticed that too but I wanted to compile only Gazebo first to make sure that works and then remove the binary packages and compile with the sources of all the other dependencies.